npm run dev 报错

```
npm init electron-vue-next testv
cd testv
npm install
npm run dev
```
上面一通操作之后，报错如下

```

> testv@0.0.0 dev /Users/alexchen/Downloads/testv
> node scripts/dev.js

/Users/alexchen/Downloads/testv/scripts/rollup.config.js:200
      await this._finishDeferred?.promise;
                                 ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:1053:16)
    at Module._compile (internal/modules/cjs/loader.js:1101:27)
    at Object.require.extensions.<computed> [as .js] (/Users/alexchen/Downloads/testv/node_modules/rollup/dist/shared/loadConfigFile.js:513:20)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at loadConfigFromBundledFile (/Users/alexchen/Downloads/testv/node_modules/rollup/dist/shared/loadConfigFile.js:521:42)
    at getDefaultFromTranspiledConfigFile (/Users/alexchen/Downloads/testv/node_modules/rollup/dist/shared/loadConfigFile.js:505:12)
    at async loadConfigFile (/Users/alexchen/Downloads/testv/node_modules/rollup/dist/shared/loadConfigFile.js:483:15)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! testv@0.0.0 dev: `node scripts/dev.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the testv@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alexchen/.npm/_logs/2021-02-12T13_46_18_368Z-debug.log
```

--- a/electron-vue-next/scripts/rollup.typescript.plugin.js
+++ b/electron-vue-next/scripts/rollup.typescript.plugin.js
@@ -86,7 +86,7 @@ export class WatchProgramHelper {
         this._finishDeferred = null
       }
 
-      await this._finishDeferred?.promise
+      await this._finishDeferred.promise
     }
   }
 }

